Sources/Public/Animation/LottieAnimationLayer.swift

```diff
@@ -728,11 +728,15 @@ public class LottieAnimationLayer: CALayer {
   }
 
   /// Sets the speed of the animation playback. Defaults to 1
-  public var animationSpeed: CGFloat = 1 {
-    didSet {
-      updateInFlightAnimation()
+    public var animationSpeed: CGFloat = 1 {
+        didSet {
+            // If a previous animation was queued but not played due to zero speed, trigger it now
+            if animationContext != nil, animationSpeed != 0, !isAnimationPlaying {
```

Comment:
Why do we need the additional `if animationContext != nil, animationSpeed != 0, !isAnimationPlaying {` condition? The comment seems to talk about when we need to call `updateInFlightAnimation()`, but previously `updateInFlightAnimation()` was always called.

Is there as case where we _shouldn't_ be calling `updateInFlightAnimation()` that is now fixed by this change?